tools/sdk-generation-pipeline/pipelines/steps/parse-and-publish-result.yml

```diff
@@ -96,6 +96,18 @@ steps:
 
           prNumber=`echo ${RESULT} | jq -r '.number'`
           prLink=`echo ${RESULT} | jq -r '.html_url'`
+          if [ 10 -gt ${#prLink} ];then
```

Comment:
@zzvswxy @dw511214992 I know I mentioned https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Submit-PullRequest.ps1 in another PR did you ever file a tracking issue for using that over creating another custom implementation here?

Reply:
Yes, there is a task in Renhe's team github backlog to track this issue:
![image](https://user-images.githubusercontent.com/20081898/202827805-ec6fdd2b-dea8-4351-a4b1-11ea8f453705.png)
I will create a issue to associate it.
Sorry for the delay,  preparing the inner bugbash takes a lot of Wei Dong and my time.

Reply:
@weshaggard Zhou is working on replacing it with `create-pull-request.yaml` , and this PR is only a hotfix and makes sure the whole workflow can work as expected. Thanks

Reply:
Thanks for the information I mostly wanted to be sure it was tracked. Yes calling Submit-PullRequest.ps1 directly give a little more flexibility and will be fine for this context.